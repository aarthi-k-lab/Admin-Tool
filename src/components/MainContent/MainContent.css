.main-content {
  align-items: stretch;
  display: flex;
  flex-grow: 1;
  flex-direction: column;
  max-width: 96%;
}

.main-content-expanded {
  align-items: stretch;
  display: flex;
  flex-grow: 1;
  flex-direction: column;
  max-width: 100%;
}
