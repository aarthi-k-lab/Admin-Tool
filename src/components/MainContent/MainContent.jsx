import React from 'react';
import PropTypes from 'prop-types';

import './MainContent.css';

const MainContent = ({ children }) => (
  <article styleName="main-content">
    { children }
  </article>
);

MainContent.propTypes = {
  children: PropTypes.node.isRequired,
};

export default MainContent;
