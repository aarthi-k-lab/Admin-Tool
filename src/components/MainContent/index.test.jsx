import React from 'react';
import renderer from 'react-test-renderer';
import MainContent from '.';

let defaultProps ={
  expandView : false,
}
describe('<MainContent />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<MainContent {...defaultProps}>content </MainContent>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
