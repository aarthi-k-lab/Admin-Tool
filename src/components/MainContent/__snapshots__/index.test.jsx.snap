// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContent /> snapshot test 1`] = `
<article
  className="components-MainContent-MainContent__main-content"
>
  content 
</article>
`;
